### ISSUE TITLE
For new person

### ISSUE THREAD

**Issue by SanjayMSD** [ram-ch/RealTimeFaceRecognition#1]
when a new person then i have to run facetrainer.py every time so i want to know that is there any way to add new person

**Comment by pranavdurai10** [comment]
I don't think so. Training has to be done in-order for the model to learn features from the newly added person's face. Hope this helps.
